hw/ip/otbn/util/build.sh

```diff
@@ -27,6 +27,6 @@ OTBN_UTIL_DIR=$(dirname "$SCRIPT_FILE")
 
 $OTBN_UTIL_DIR/otbn-as $1 -o $2.o
 $OTBN_UTIL_DIR/otbn-ld $2.o -o $2.elf
-$OTBN_UTIL_DIR/otbn-objdump -fhSD $2.elf > $2.dis
+$OTBN_UTIL_DIR/otbn-objdump -f -h -S -D $2.elf > $2.dis
```

Comment:
Ooh, that's a point. Please could you file an issue to remind us to fix this properly? The aim is for `otbn-objdump` to behave like any other objdump (there's an "am I disassembling?" check, which looks for `-d` or `-D`, but won't fire on `-foobarD`).